tests/import_test.go

```diff
@@ -0,0 +1,30 @@
+package tests_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"kubevirt.io/containerized-data-importer/tests/framework"
+)
+
+const (
+	testSuiteName   = "Importer Test Suite"
+	namespacePrefix = "importer"
+)
+
+var _ = Describe(testSuiteName, func() {
+	f := framework.NewFramework(namespacePrefix)
+
+	Describe("Creating standard PVC should not trigger importer", func() {
+		It("PVC should remain empty after being created", func() {
+			pvc, err := f.CreatePVCFromDefinition(f.Namespace.GetName(), framework.NewPVCDefinition("no-import", "1G", nil, nil))
+			Expect(err).ToNot(HaveOccurred())
+			// Wait a while to see if CDI puts anything in the PVC.
+			time.Sleep(framework.DefaultTimeout)
```

Comment:
need specific timeout names

Reply:
See my comment on the specific vs generic timeout names. If we want specific timeouts for an entity we can do that, but I think its useful to have a generic timeout.